ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/embeddedActivityBlock/embeddedActivityBlock.component.ts

```diff
@@ -60,9 +60,16 @@ export class EmbeddedActivityBlockComponent implements OnInit {
     }
 
     public openDeleteDialog(): void {
-        const config = this.modalService.getDeleteDialogConfig(this.deleteButtonRef);
+        const panelClass = 'modal-activity-block__delete-dialog';
+        const config = this.modalDialogService.getDialogConfig(this.deleteButtonRef, panelClass);
+        config.data = {
+            title: 'SDK.ConfirmDeletion',
+            confirmBtnText: 'SDK.DeleteButton',
+            cancelBtnText: 'SDK.CancelBtn',
+            confirmBtnColor: 'warn'
+        };
 
-        const dialogRef = this.dialog.open(ActivityDeleteDialogComponent, config);
+        const dialogRef = this.dialog.open(ConfirmDialogComponent, config);
         dialogRef.afterClosed().subscribe((confirmDelete: boolean) => {
```

Comment:
Do we have unsubscribe from dialog events? (Highlighting this instance but there are a couple other ones in file)

Reply:
we don't need to unsubscribe from afterClosed result since the observable itself completes